asset-tests/app/src/androidTest/java/org/gearvrf/tester/AssetImportTests.java

```diff
@@ -528,9 +529,31 @@ public void jassimpSkinningTREX() throws TimeoutException
     }
 
     @Test
-    public void jassimpBoxGLTF() throws TimeoutException
+    public void jassimpRoughBoomboxGLTF() throws TimeoutException
     {
-        mHandler.loadTestModel(GVRTestUtils.GITHUB_URL + "jassimp/gltf/BoxTextured-glTF/BoxTextured.gltf", 0, 2, "jassimpBoxGLTF");
+        GVRSceneObject light1 = createLight(mTestUtils.getGvrContext(), 1, 1, 1, 1.8f);
+        GVRSceneObject light2 = createLight(mTestUtils.getGvrContext(), 1, 1, 1, -0.8f);
+        mRoot.addChildObject(light1);
+        mRoot.addChildObject(light2);
+
+        mHandler.loadTestModel(GVRTestUtils.GITHUB_URL + "/jassimp/gltf/BoomBox/BoomBox.gltf", 5, 1, "jassimpRoughBoomboxGLTF");
```

Comment:
I don't think BoomBox.gltf is at that location. I see it under the app/assets folder.